src/database/schemas/address.schema.ts

```diff
@@ -0,0 +1,59 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import { CountryEnum } from '../enums/country.enum';
+
+export type AddressType = Address & Document;
+
+@Schema({ _id: false })
+export class Address {
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 256,
+    })
+    street: string;
+
+    @Prop({
+        type: String,
+        trim: true,
+        lowercase: true,
+        maxLength: 8,
+    })
+    streetNumber: string;
```

Comment:
question(blocking): Are there street numbers that reach 9999 9999?

Reply:
google says the highest streetnumber is 6 digits since we wanted to use potencies of 2 it should be 8. Should we stop with the potencies?